### ISSUE TITLE
`om config-template` missing syslog feature for spring-cloud-services

### ISSUE THREAD

**Issue by jgeorge300** [pivotal-cf/om#485]
Running the config-template command against Spring Cloud Services v3.1 does not generate feature files for the syslog configuration. Running staged-config shows the missing properties.

```
$ om config-template --output-directory templates/ --pivnet-product-slug p-spring-cloud-services --product-version '3.1.7' --pivnet-api-token ${PIVNET} --file-glob '*.pivotal'
$ ls -l templates/p_spring-cloud-services/3.1.7/features/
$
```

Expected: A features file and properties similar to the MySQL and RabbitMQ tiles to configure syslog

**Comment by kcboyle** [comment]
Hi @jgeorge300, just to clarify so I know which label to add to this issue, you are not currently _blocked_ and were able to find the workaround through `staged-config`, correct?

**Comment by kcboyle** [comment]
Adding PR welcome label. `config-template` is still marked `EXPERIMENTAL`, and might not get worked on right away. However, feel free to explore the codebase, test, and submit a PR to help fix the issue if you want/need it sooner!

**Comment by jgeorge300** [comment]
Yes, I have a work around for the time being.

**Comment by kcboyle** [comment]
@jgeorge300 
Can you clarify something for me, real quick? 
Does the syslog ops file exist for mysql and rabbit, but _not_ for spring cloud?

**Comment by jtarchie** [comment]
It looks like the RabbitMQ tile and MySQL tile are not using the OpsManager built-in Syslog support.
They are providing their own by bundling the syslog bosh release.
They define blueprint properties like so:

```yaml
- configurable: true
  default: enabled
  name: syslog_selector
....
```

That's why config template generates them

The spring cloud services tile set `opsmanager_syslog: true`.
The config template command does not understand what to do with this feature.

**Comment by jgeorge300** [comment]
@kcboyle correct

**Comment by kcboyle** [comment]
Work has been done for this. It will be released in `om 4.7.0`
